qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -78,20 +78,32 @@ def run(self, qobj):
 
         return job
 
-    def properties(self, refresh=False):
-        """Return the online backend properties.
+    def properties(self, refresh=False, datetime_filter=None):
+        """Return the online backend properties with optional filtering.
 
         Args:
             refresh (bool): if True, the return is via a QX API call.
                 Otherwise, a cached version is returned.
+            datetime_filter (datetime.datetime): by specifying a filter,
+                this function returns an instance of the BackendProperties whose
+                timestamp is closest to, but older than the specified datetime
+                filter.
 
         Returns:
-            BackendProperties: The properties of the backend.
+            BackendProperties: The properties of the backend. If the backend has
+                no properties to display, it returns ``None``.
         """
         # pylint: disable=arguments-differ
+        api_filter = {}
+        if datetime_filter:
+            api_filter['last_update_date'] = {'lt': datetime_filter.isoformat()}
+
         if refresh or self._properties is None:
-            api_properties = self._api.backend_properties(self.name())
-            self._properties = BackendProperties.from_dict(api_properties)
+            api_properties = self._api.backend_properties(self.name(), api_filter=api_filter)
+            if not api_properties:
+                self._properties = None
```

Comment:
As informally discussed, can you make sure that if the argument is specified, no modification of `self._properties` or any functionality related to "caching" is involved?

The caching was introduced as asking for the properties of a backend happens a number of times during a typical Qiskit program (without the user explicitly asking for them in a number of cases), and it became a bit expensive to make an API call each time. However, in this case let's assume that asking for the properties of a backend _at a specified timestamp_ is a more specific operation, and fully under the control of the user, and he will be ok assuming the "cost" of the call each time (or that it is their responsibility to do some intelligent caching if needed).